### ISSUE TITLE
 m_webView.LoadUrl() Issue

### ISSUE THREAD

**Issue by krishnanpc** [TLabAltoh/TLabWebViewVR#33]
Thanks for the wonderful plugin!

I have an issue, every time I call  m_webView.LoadUrl(), defaults to the original URL set in the inspector.

Is this a known issue? Perhaps something to do with permissions?

**Comment by TLabAltoh** [comment]
Thanks for the report. Is the problem like below?

<details><summary>problem 1</summary>

```cs
private TLabWebView m_webview; // (https://youtube.com seted as default in inspector)

m_webview.LoadUrl("https://google.com"); 
// --> Load https://youtube.com

m_webview.LoadUrl("https://doc.unity3d.com"); 
// --> Load https://youtube.com
```

</details>

<details><summary>problem 2</summary>

```cs
private TLabWebView m_webview; // (https://youtube.com seted as default in inspector)

m_webview.LoadUrl("https://google.com"); 
Debug.Log(m_webview.url);
// --> "https://youtube.com"

m_webview.LoadUrl("https://doc.unity3d.com"); 
Debug.Log(m_webview.url);
// --> "https://youtube.com"
```

</details>

Both is a bug in the plugin, but it should have been fixed in the latest update (```v0.0.10+```). Please try a newer version of the plugin with the following command.

```bat
@rem cd REPOSITORY_ROOT
git fetch origin master
git pull origin master

git submodule update --init
```

**Comment by krishnanpc** [comment]
Hey, thank you so much for the quick reply and resolution! :) 
Many cheers and have a good day/evening.
